src/providers/app-config/app-config.ts

```diff
@@ -26,7 +26,12 @@ export class AppConfigProvider {
   public loadRemoteConfig = (): Promise<any> =>
     this.getRemoteData()
       .then(data => this.mapRemoteConfig(data))
-      .catch(error => console.log('Error Getting Remote Config', error))
+      .catch((error) => {
+        if (error && error.status === 403) {
+          return Promise.reject('user not authorised');
```

Comment:
Can we use the constant from `src/providers/authentication/authentication.constants.ts`?

Reply:
Using the constants file now